app/src/main/java/com/github/se/travelpouch/ui/dashboard/TravelActivityScreen.kt

```diff
@@ -0,0 +1,184 @@
+import com.github.se.travelpouch.model.activity.ActivityModelView
+import com.github.se.travelpouch.ui.navigation.NavigationActions
+import com.google.android.material.bottomnavigation.BottomNavigationItemView
+import com.google.android.material.bottomnavigation.BottomNavigationMenuView
+import com.google.android.material.bottomnavigation.BottomNavigationView
+import com.google.android.material.navigation.NavigationBarItemView
+import com.google.firebase.Timestamp
+import java.util.Calendar
+import java.util.GregorianCalendar
+
+data class BottomNavigationItem(
+    val title: String,
+    val icon: ImageVector
+)
+
+@Preview
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun TravelActivitiesScreen(
+    navigationActions: NavigationActions? = null,
+    activityModelView: ActivityModelView? = null
+) {
+
+    val activites_test = listOf(
+        Activity(
+            "1",
+            "title1",
+            "description1",
+            Location(
+                0.0,
+                0.0,
+                Timestamp(0, 0),
+                "lcoation1"
+            ),
+            Timestamp(0, 0),
+            mapOf<String, Int>()
+        ),
+
+        Activity(
+            "2",
+            "title2",
+            "description2",
+            Location(
+                0.0,
+                0.0,
+                Timestamp(0, 0),
+                "lcoation2"
+            ),
+            Timestamp(0, 0),
+            mapOf<String, Int>()
+        )
+    )
+    //val listOfActivities = activityModelView.activities.collectAsState()
+    val listOfDestinations = listOf(
+        BottomNavigationItem("Activities", Icons.Default.Home),
+        BottomNavigationItem("Map", Icons.Default.Place)
+    )
+
+    Scaffold(
+        modifier = Modifier.testTag("travelActivitiesScreen"),
+        topBar = {
+            TopAppBar(
+                title = { Text("Travel", Modifier.testTag("travelTitle")) },
+                navigationIcon = {
+                    IconButton(onClick = {}, modifier = Modifier.testTag("goBackButton")) {
+                        Icon(
+                            imageVector = Icons.AutoMirrored.Outlined.ArrowBack,
+                            contentDescription = "Back")
+                    }
+                },
+                actions = {
+                    IconButton(onClick = {}) {
+                        Icon(
+                            imageVector = Icons.Default.Settings,
+                            contentDescription = null
+                        )
+                    }
+
+                    IconButton(onClick = {}){
+                        Icon(
+                            imageVector = Icons.Default.DateRange,
+                            contentDescription = null
+                        )
+                    }
+                })
+        },
+
+        bottomBar = {
+            NavigationBar {
+                listOfDestinations.forEach { NavigationBarItem(
```

Comment:
I don't understand why you use a navigation bar?

Reply:
Well, it is more like a feature because I wanted to have two screens, one that displays the list of all the activities to do, and another that marks the activity of a travel on a map. And we could navigate between the two screens.

Reply:
yes I understand